infra/infra.go

```diff
@@ -47,6 +47,10 @@ func main() {
 		Short('a').
 		StringVar(&g.Auth)
 
+	k8sGKEShow := k8sGKE.Command("info", "show GKE related infomation")
+	k8sGKEShow.Command("deploymentvars", "gke info deploymentvars").
+		Action(dr.GetDefaultDeploymentVars)
```

Comment:
We probably should have only one command `infra gke info`, we can call the `GetDefaultDeploymentVars` action for now, but later we can extend it to print out other useful things :)

Reply:
Done